ApiTests/UserAuthenticationTests.cs

```diff
@@ -43,6 +43,10 @@ public void GetServerSettings()
             _clientId = Environment.GetEnvironmentVariable("TSHEETS_CLIENTID");
             _redirectUri = Environment.GetEnvironmentVariable("TSHEETS_REDIRECTURI");
             _clientSecret = Environment.GetEnvironmentVariable("TSHEETS_CLIENTSECRET");
+
+            Assert.IsNotNull(_clientId);
```

Comment:
This isn't part of the fix, but it's helpful if you forget to set your environment as the tests will fail in a somewhat obscure way.